src/util/rocksdb/mod.rs

```diff
@@ -341,6 +341,33 @@ pub fn compact_range(
     db.compact_range_cf_opt(handle, &compact_opts, start_key, end_key);
 }
 
+pub fn compact_files(db: &DB, cf: &CFHandle, output_level: u32) -> Result<(), String> {
```

Comment:
do we need to move this to the RocksDB repo?

Reply:
I don't think so, this is just a specific use case for us, like other RocksDB related code here.